README.md

```diff
@@ -1,51 +1,58 @@
-hosted at [telemetry.mozilla.org](http://telemetry.mozilla.org).
 
-But the aggregated data is also available for consumption by third-party
-applications, so you don't need to do the aggregation on your own.
+This repository contains the **source code for [telemetry.mozilla.org](http://telemetry.mozilla.org)**. The dashboards on this site can be used for everything from checking measure values to figuring out common causes of Firefox hangs.
 
-Consuming Telemetry Aggregations
---------------------------------
-Include into your code `http://telemetry.mozilla.org/v1/telemetry.js` feel free
-to use the other modules too. Don't go about reading the raw JSON files,
-they are not designed for human consumption!
+The main dashboards on [telemetry.mozilla.org](http://telemetry.mozilla.org) consume data from Telemetry's v2 and v4 backends using Telemetry.js. For an overview of how all these systems fit together, see [Telemetry Demystified](http://anthony-zhang.me/blog/telemetry-demystified/).
 
-We recommend that you include `telemetry.js` from
-`http://telemetry.mozilla.org/v1/telemetry.js`, this file is used to access all
-the generated aggregates. We will undertake reasonable effort to ensure that it
-remains backwards compatible and continously updated with new aggregates.
+The dashboards that do not use Telemetry.js generally use [scheduled analysis jobs](http://telemetry-dash.mozilla.org/) that regularly publish data on S3. The source code for these can be found in their respective repositories.
 
-Please refer to the [official documentation](http://telemetry.mozilla.org/docs.html)
-for `telemetry.js` for instructions on how to use this library.
+This repository also contains the **source code for Telemetry.js** (all versions). The specific files can be found under the `v1/` and `v2/` directories.
 
-  * [telemetry.js documentation](http://telemetry.mozilla.org/docs.html)
+We currently have two versions of the Histogram Dashboard and Evolution Dashboard - one for the v2 pipeline, and one for the v4 pipeline. The v4 versions live in the `new-pipeline/` directory. The v2 versions live in the project root.
+
+Deploying Telemetry Dashboard
+-----------------------------
+
+The [telemetry.mozilla.org](http://telemetry.mozilla.org) site is hosted on S3, in the [telemetry.mozilla.org bucket](https://console.aws.amazon.com/s3/home#&bucket=telemetry.mozilla.org). In front of S3, there is also the CloudFront CDN (managed by :mostlygeek).
+
+The contents of this bucket are uploaded from the telemetry dashboard master node instance on AWS. The currently active dashboard master node is `ec2-54-202-211-22.us-west-2.compute.amazonaws.com`. On the node, there is a script `~/update-telemetry.mozilla.org.sh`, run by cron every day, that pulls the latest master for telemetry-dashboard from the GitHub repository and uploads it to the S3 bucket using the `aws s3 sync . s3://telemetry.mozilla.org/ --delete --region us-east-1 --exclude ".*"` command.
```

Comment:
how i find the currently active node? in the AWS dash?
where does update-telemetry.mozilla.org.sh live? can it be in github (are there credentials etc)?